tests/TestCase.php

```diff
@@ -13,8 +12,6 @@ class TestCase extends PHPUnit
     public function setUp(): void
     {
         parent::setUp();
-
-        $dotenv = Dotenv::createImmutable(dirname(__DIR__));
-        $dotenv->safeLoad();
+        // Do something which should run for each test file (if needed), env variable loading is moved to bootstraping
```

Comment:
do we need this comment here? and in fact, if there is nothing in the setup method, maybe we don't need to define it at all

Reply:
Lets keep a base `TestCase` file, will remove comment